templates/package.json

```diff
@@ -4,7 +4,8 @@
   "private": "true",
   "license": "MIT",
   "scripts": {
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "preversion": "node ./scripts/sync-up-version.js && git add ."
```

Comment:
this preversion script run when npm run lerna version only if templates has code change or templates is force-publish